docs/src/pages/components/date-picker/date-picker.md

```diff
@@ -45,10 +45,10 @@ The date picker is rendered as a modal dialog on mobile, and a textbox with a po
 
 The date picker component is designed and optimized for the device it runs on.
 
-- The "Mobile" version works best for touch devices and small screens.
-- The "Desktop" version works best for mouse devices and large screens.
+- The `MobileDatePicker` component works best for touch devices and small screens.
+- The `DesktopDatePicker` component works best for mouse devices and large screens.
 
-By default, the `DatePicker` component uses a `@media (pointer: fine)` media query to determine which version to use.
```

Comment:
It's used, ok. But whether it renders the desktop version with a fine pointer or the mobile version is not mentioned. Though one should be safe to assume fine pointer implies desktop. But let's not encourage readers to assume our implementation.